website/content/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title : "GDG - Grafana Dash-n-Grab"
+description: "GDG is a tool used to manage dashboards, connections, orgs and various entities of the Grafana application."
+lead: "GDG is a tool used to manage dashboards, connections, orgs and various entities of the Grafana application."
+date: 2020-10-06T16:33:54+02:00
```

Comment:
A lot of these dates probably need updating.

Reply:
Yeah, the only one that really matters are the docs, and those are updated.